include/Options/Options.php

```diff
@@ -356,9 +361,11 @@ public function get_schema(): array {
 			'title'                => static::OPTION_NAME,
 			'description'          => __( 'Polylang options', 'polylang' ),
 			'type'                 => 'object',
-			'context'              => array( 'edit' ),
+			'context'              => array( 'view', 'edit', 'embed' ),
+			'default'              => $default,
```

Comment:
Why did you add the `default` values all at once at the top level instead of having a `default` key on each sub-instances?
Regarding the spec it seems doable, [see](https://json-schema.org/draft/2020-12/draft-bhutton-json-schema-validation-00#rfc.section.9.2).

Reply:
Not "instead", it is also done [in each sub-option](https://github.com/polylang/polylang/pull/1569/files#diff-271c92425f5047de724b1a26c3011a77ba0a637d63e23c0ba7cb61816093c90cR160). This one is in case of the whole option is considered not valid.